### PR TITLE
feat(utility-classes): flex section

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Adding additional utility classes related to flex. 
Those are copied over from react demo environment.

**Solving issue**  
Fixes: [CDEP-2180](https://tegel.atlassian.net/browse/CDEP-2180)

**How to test**  
1. Check _utility-classes file
2. Check if the same content is under Storybook story for utility classes under "Flex"
3. Feel free to suggest more if you feel is needed in this point of time

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

[CDEP-2180]: https://tegel.atlassian.net/browse/CDEP-2180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/src/global/_utility-classes.scss [hunk 1]
```diff
@@ -380,3 +380,40 @@
...
+  gap: 16px !important;
+}
+
+.tds-u-flex-gap3 {
+  gap: 24px !important;
+}
+
+.tds-u-flex-gap4 {
+  gap: 32px !important;
+}
```
Comment (mJarsater) [inline core/src/global/_utility-classes.scss:419]:
```suggestion
.tds-u-gap1 {
  gap: 8px !important;
}

.tds-u-gap2 {
  gap: 16px !important;
}

.tds-u-flex-gap3 {
  gap: 24px !important;
}

.tds-u-gap4 {
  gap: 32px !important;
}
```

File: core/src/stories/utility/utility-classes/flex.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  gap: 16px !important;
+}
+
+.tds-u-flex-gap3 {
+  gap: 24px !important;
+}
+
+.tds-u-flex-gap4 {
+  gap: 32px !important;
+}
```
Comment (nathalielindqvist) [inline core/src/stories/utility/utility-classes/flex.stories.mdx:37]:
```suggestion
.tds-u-gap1 {
  gap: 8px !important;
}

.tds-u-gap2 {
  gap: 16px !important;
}

.tds-u-gap3 {
  gap: 24px !important;
}

.tds-u-gap4 {
  gap: 32px !important;
}
```

To match updated classes in scss file

File: core/src/global/_utility-classes.scss [hunk 3]
```diff
@@ -380,3 +380,40 @@
...
+  gap: 8px !important;
+}
+
+.tds-u-gap2 {
+  gap: 16px !important;
+}
+
+.tds-u-flex-gap3 {
+  gap: 24px !important;
+}
```
Comment (nathalielindqvist) [inline core/src/global/_utility-classes.scss:415]:
```suggestion
.tds-u-gap3 {
  gap: 24px !important;
}
```

To match other classes

**Comment by theJohnnyMe** [comment]
@nathalielindqvist  - Good that you checked this, I missed mdx file! <3

**Review by nathalielindqvist — APPROVED** [review]
Looks good!

**Pull request merged**